### PR TITLE
rockpi4: Add rockpi4 support

### PR DESCRIPTION
**Pull request by brian541**
hi 
add radxa [rockpi4](https://wiki.radxa.com/Rockpi4) model A/B boards support

### CODE REVIEW

File: include/arm/rockpi4.h [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/*
+ * Author: Thomas Ingleby <thomas.c.ingleby@intel.com>
+ * Author: Michael Ring <mail@michael-ring.org>
+ * Copyright (c) 2014 Intel Corporation.
```
[the remark this answers is not in the document]
Comment (brian541) [reply include/arm/rockpi4.h]: I'm sorry i forgot this,and i modify some place you suggest,please have a look again,thanks so much

**Comment by tingleby** [comment]
OK Great. 
Could you please use a full name in the sign-off please.

i.e. 
Signed-off-by: Random J Developer <random@developer.example.org>
Using your real name (sorry, no pseudonyms or anonymous contributions.)

**Comment by brian541** [comment]
> .
Sorry to reply you late.
is that ok?

**Comment by tingleby** [comment]
Perfect, thanks for the commit!

**Pull request closed without merging**